ansible/roles/datadog/templates/kafka_consumer.yaml.j2

```diff
@@ -0,0 +1,35 @@
+init_config:
+#  Customize the ZooKeeper connection timeout here
+  zk_timeout: 5
+#  Customize the Kafka connection timeout here
+  kafka_timeout: 5
+
+instances:
+  - kafka_connect_str: localhost:9092  # Should always be deployed to broker so localhost is OK
+    zk_connect_str: "{{ groups['zookeeper'][0] }}:{{ zookeeper_client_port }}"
+    zk_prefix: /0.8
+    consumer_groups:
+      'form-processsor':
```

Comment:
As far as I know the groups are what gets passed to the `KafkaChangeFeed` class as `group_id`. e.g.
```
KafkaChangeFeed(topics=[GROUP], group_id='groups-to-es')
```